docs/features/network-security-controls/network-policy.md

```diff
@@ -58,7 +178,107 @@ spec:
   policyTypes:
   - Egress
 ```
-**OVN-Implementation:**
+
+#### **Selector Based Denial**
+
+We can extend the default denial behavior by using the `podSelector` field to selectively block all traffic from/to certain pods, in this case blocking all unicast traffic from reaching apps labeled with demo.
+
+```
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: default-deny-all
+spec:
+  podSelector:
+    matchLabels:
+          app: demo
+  policyTypes:
+  - Ingress
+  - Egress
+```
+
+#### **Selector Based filtering**
+
+The `from` selector for an ingress and the `to` selector for an egress can be used to select particular pods or namespaces that will be allowed as destinations or sources of data, as in the example below, which allows all incoming traffic from the namespace db_app into the networkPolicy namespace.
```

Comment:
`from` and `to` are not selectors, they are lists of rules, that in turn may be selector-based or ipBlocks.